.ado/continuous.yml

```diff
@@ -21,7 +21,7 @@ stages:
 
           - template: templates/yarn-install.yml
 
-          - script: npx --no-install beachball bump --branch origin/$(Build.SourceBranchName) --yes
+          - script: npx beachball bump --branch origin/$(Build.SourceBranchName) --yes
```

Comment:
The behavior of the new `npx` is to prefer the local version, and to prompt if it needs to fetch something. We change the configuration from prompting to instead immediately failing in CI, since it is non-interactive. So, anything remote needs a `--yes` attached to it.

There are a couple of other differences between `yarn` vs `npx` to run a local package. E.g. `yarn` will change the `cwd` to be that of the package root, no matter where you run the script (which is problematic for file path args...).